### DISCUSSION TITLE
How to configure I18n?

### DISCUSSION THREAD

**Discussion by dimas-cyriaco** [rubymonolith/superform#45] [Q&A]
The default Rails form builder calls I18n helpers to translate model name and attribute names by default.

For example, for the submit button text of an update form, it tries `I18n.translate('helpers.submit.update')` (among others).

Can superform be configured to do the same?

**Comment by janpeterka** [comment]
I did manage to do it this way:
```ruby
   def row(component)
      translation = t("activerecord.attributes.#{component.field.parent.object.class.name.underscore}.#{component.field.key}")

      div do
        render component.field.label { translation }
        render component
      end
    end
 ```
